clang/lib/DPCT/DPCT.cpp

```diff
@@ -843,6 +855,17 @@ int runDPCT(int argc, const char **argv) {
 
   std::vector<std::string> SourcePathList;
   if (QueryAPIMapping.getNumOccurrences()) {
+    if (QueryAPIMapping.getNumOccurrences() > 1) {
+      llvm::outs() << "Warning: Each query focuses on one API. Only the last "
+                      "one will be queried.\n";
+    }
+    if (hasOptConflictWithQuery(argc, argv)) {
+      llvm::outs()
+          << "Warning: When using --query-api-mapping, only "
```

Comment:
"Warning: For API mapping query function triggered by option --query-api-mapping, all other option are ignored except "--extra-arg" and "--cuda-include-path".

Reply:
Now the logic is just exit here but not ignore, if we need ignore, we need clear some option variable here be cause the `CommonOptionsParser::create` has done.